keys.go

```diff
@@ -78,10 +77,10 @@ func (p *Parser) decodeString(key string, value string) (err error) {
 			repo1 := vcsurl.GetRepo(url1)
 			repo2 := vcsurl.GetRepo(url2)
 			if repo1 == nil {
-				log.Printf("Warning: go-vcsurl failed to detect repo for %s\n", url1.String())
```

Comment:
@bfabio why was this considered a non-blocking warning?

Reply:
My guess is that it was just a mistake and we just never triggered it.